template/vtex/src/components/Example/index.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
```

Comment:
this file should be in root of `src` folder, the components folder no is required for us.

Reply:
Fixed on commit 9f472c8.